build/mac/pkg-scripts/postinstall

```diff
@@ -28,7 +28,7 @@ fi
 if [ "$promoCode" == "" ]; then
   appName=$(basename "$1" | sed "s/.pkg//" | sed "s/ /\-/g")
 else
-  appName=$(basename "$1" | sed "s/.pkg//" | sed "s/ /\-/g" | sed "s/-${promoCode}//")
+  appName=$(basename "$1" | sed "s/.pkg//" | sed "s/ /\-/g" | perl -ple "s/-${promoCode}//i")
```

Comment:
is perl guaranteed to be installed on people's machines? (base macOS install)?

Reply:
Based on what perl.org is saying on this page it seems so: https://www.perl.org/get.html